scripts/mergedEquippableUserJourney.ts

```diff
@@ -227,196 +227,166 @@ async function mintTokens(
   console.log("Nest-minting Gem tokens");
   let allTx: ContractTransaction[] = [];
   for (let i = 1; i <= totalBirds; i++) {
-    let tx = await gem.mintNesting(kanaria.address, 3, i, {
+    let tx = await gem.nestMint(kanaria.address, 3, i, {
       value: pricePerMint.mul(3),
     });
     allTx.push(tx);
   }
   await Promise.all(allTx.map((tx) => tx.wait()));
   console.log(`Minted 3 gems into each kanaria`);
 
-  // Accept 1 gem for each kanaria
+  // Accept 3 gems for each kanaria
   console.log("Accepting Gems");
-  for (let i = 0; i < 3; i++) {
-    allTx = [];
-    for (let tokenId = 1; tokenId <= totalBirds; tokenId++) {
-      let tx = await kanaria.acceptChild(tokenId, 0);
-      allTx.push(tx);
-    }
-    await Promise.all(allTx.map((tx) => tx.wait()));
-    console.log(`Accepted 1 gem for each kanaria`);
+  for (let tokenId = 1; tokenId <= totalBirds; tokenId++) {
+    allTx = [
+      await kanaria.acceptChild(tokenId, 2, gem.address, 3 * tokenId),
+      await kanaria.acceptChild(tokenId, 1, gem.address, 3 * tokenId - 1),
+      await kanaria.acceptChild(tokenId, 0, gem.address, 3 * tokenId - 2),
+    ];
   }
+  await Promise.all(allTx.map((tx) => tx.wait()));
+  console.log(`Accepted gems for each kanaria`);
 }
 
-async function addKanariaResources(
+async function addKanariaAssets(
   kanaria: SimpleEquippable,
   baseAddress: string
 ): Promise<void> {
-  console.log("Adding Kanaria resources");
-  const resourceDefaultId = 1;
-  const resourceComposedId = 2;
+  console.log("Adding Kanaria assets");
+  const assetDefaultId = 1;
+  const assetComposedId = 2;
   let allTx: ContractTransaction[] = [];
-  let tx = await kanaria.addResourceEntry(
-    {
-      id: resourceDefaultId,
-      equippableGroupId: 0, // Only used for resources meant to equip into others
-      baseAddress: ethers.constants.AddressZero, // base is not needed here
-      metadataURI: "ipfs://default.png",
-    },
+  let tx = await kanaria.addAssetEntry(
+    0, // Only used for assets meant to equip into others
+    ethers.constants.AddressZero, // base is not needed here
+    "ipfs://default.png",
     [],
     []
   );
   allTx.push(tx);
 
-  tx = await kanaria.addResourceEntry(
-    {
-      id: resourceComposedId,
-      equippableGroupId: 0, // Only used for resources meant to equip into others
-      baseAddress: baseAddress, // Since we're using parts, we must define the base
-      metadataURI: "ipfs://meta1.json",
-    },
+  tx = await kanaria.addAssetEntry(
+    0, // Only used for assets meant to equip into others
+    baseAddress, // Since we're using parts, we must define the base
+    "ipfs://meta1.json",
```

Comment:
Here I go again with the keys:

```suggestion
    {
        equippableGroupId: 0, // Only used for assets meant to equip into others
        baseAddress: baseAddress, // Since we're using parts, we must define the base
        metadataURI: "ipfs://meta1.json",
    }
```